python/metatensor-torch/metatensor/torch/atomistic/ase_calculator.py

```diff
@@ -264,12 +264,12 @@ def calculate(
             if "stress" in properties:
                 do_backward = True
 
-                scaling = torch.eye(3, requires_grad=True, dtype=self._dtype)
+                strain = torch.eye(3, requires_grad=True, device=positions.device, dtype=self._dtype)
```

Comment:
```suggestion
                strain = torch.eye(3, requires_grad=True, device=self._device, dtype=self._dtype)
```

This should be the same one, but convey intent a bit clearer